plat/mediatek/mt6795/aarch64/platform_common.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
The filename suggests a common file to Mediatek platforms, but there is one for each platform and part of the code is duplicated. Could this file be placed in plat/mediatek/common and shared with mt8173?